File: 2-semestre/uri-online-judge/c/1175.c
#include <stdio.h>
 
int main() {
    int arr[20], i, j;

    for (i = 19; i >= 0; i--) {
        scanf("%d", &arr[i]);
    }

    for (j = 0; j < 20; j++) {
        printf("N[%d] = %d\n", j, arr[j]);
    }

    return 0;
}

File: 2-semestre/uri-online-judge/javascript/1175.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

let arr = [];

for (let i = 19; i >= 0; i--) {
    arr[i] = parseInt(lines.shift());
}

for (let j = 0; j < 20; j++) {
    console.log(`N[${j}] = ${arr[j]}`);
}
